content/browser/storage_partition_impl.cc

```diff
@@ -536,7 +538,11 @@ StoragePartitionImpl* StoragePartitionImpl::Create(
   StoragePartitionImpl* storage_partition = new StoragePartitionImpl(
       context, partition_path, quota_manager.get(), appcache_service.get(),
       filesystem_context.get(), database_tracker.get(),
+#ifndef DISABLE_INDEXEDDB
       dom_storage_context.get(), indexed_db_context.get(),
+#else
+      dom_storage_context.get(), nullptr,
```

Comment:
In BrowserContext::SaveSessionState()
{ ...
    if (indexed_db_context_impl->TaskRunner())    ===> will cause crash here? de-reference a nullptr.
}